external/external.sh

```diff
@@ -19,6 +19,33 @@ set -e
 
 source $(dirname "$0")/provider.sh
 
+if [ -z "${EXTRA_DOCKER_ARGS}" ] ; then
+  echo \
+"   # ================= WARNING ================= WARNING ================= WARNING ================= #
+   # EXTRA_DOCKER_ARGS are not set. You will be testing java which is already in container and not TEST_JDK_HOME
+   # TEST_JDK_HOME is set to $TEST_JDK_HOME but will not be used. See test_base_functions.sh for order of search
+   # You should set your's TEST_JDK_HOME to mount to /opt/java/openjdk, eg:                          #
+   # export EXTRA_DOCKER_ARGS=\"-v \$TEST_JDK_HOME:/opt/java/openjdk\"                               #
+   # ================= WARNING ================= WARNING ================= WARNING ================= #"
+else
+  echo \
+"   # Info Info  Info Info Info Info Info Info Info Info Info Info Info Info Info Info Info Info Info #
```

Comment:
Will this whitespacing produce a consistent output?

Reply:
I had aligned itto that "====" format Shelly suggested. They provide as consistent output as possible. Do you have any more exact concern with them? Like being to long or simiarly?